config/locales/en.yml

```diff
@@ -233,6 +238,9 @@ en:
         sufficient_processing:
           Have non-originating parts been subject to sufficient processing to
           qualify for preferential treatment?
+      rules_of_origin_steps_subdivision:
+        subdivision: Provide more information about your product
```

Comment:
it looks a bit redundant to use `subdivision` under `rules_of_origin_steps_subdivision`,
at least a first sight (maybe there is a good reason)

Reply:
These are the govuk form builders default keys, not ones I'm adding - this is equivalent to

```yaml
model_name_space_plus_model_class_underscored:
  attribute_name: Legend to show for attribute
```

Reply:
I've renamed the step to `Subdivisions` and the attribute field to `subdivision_id` to make this clearer :+1: